build/source/netcdf/modelwrite.f90

```diff
@@ -604,6 +604,23 @@ subroutine writeRestart(filename,         & ! intent(in): name of restart file
  err = nf90_put_att(ncid,ncVarID(nProgVars+1),'long_name',trim(bvar_meta(iLookBVAR%routingRunoffFuture)%vardesc));   call netcdf_err(err,message)
  err = nf90_put_att(ncid,ncVarID(nProgVars+1),'units'    ,trim(bvar_meta(iLookBVAR%routingRunoffFuture)%varunit));   call netcdf_err(err,message)
 
+ ! define selected basin variables (pothole storage - HDS) -> vMin
```

Comment:
Consider generalizing this code to loop through variables in the same way that we do for HRU variables.

Reply:
Done